javascript/javascript1/week2/week2.js

```diff
@@ -0,0 +1,159 @@
+//fullname function
+function getFullname(fullname1, fullname2) {
+    return fullname1 + " " + fullname2
+}
+console.log(getFullname("Benjamin", "Hughes"))
+
+
+//Formal fullname
+function getFullname(fullname1, fullname2, useFormalName = false, gender = "male") {
+    if (useFormalName) {
+        if (gender === "male") {
+
+            return "Lord" + fullname1 + " " + fullname2
+        } else {
+            return "Miss" + fullname1 + " " + fullname2
+        }
+    } else {
+        return fullname1 + " " + fullname2
+    }
+
+}
+
+
+//Event application
+const day = ["sunday", "monday", "tuesday", "wendsday", "thursday", "friday", "saterday"]
+
+function getEventWeekday(today, eventDay) {
+    let eventDay_ = eventDay % 7;
```

Comment:
name is kind of the same as a parameter maybe is better to change it like indexOfDay